exchange/bitswap/decision/peer_request_queue.go

```diff
@@ -198,12 +266,20 @@ type activePartner struct {
 
 	// priority queue of tasks belonging to this peer
 	taskQueue pq.PQ
+
+	//  kinda dirty to give each active partner all the ledgers and it's pid
+	//  instead of just its own ledger, but i'm not sure how else to deal with
+	//  the lazily loaded ledgers
```

Comment:
so is the advantage just nicer naming?  instead of a pointer to a map and a peer id `activePartners` have a pointer to a `LedgerSet` and a peer id?